cranelift/filetests/src/zkasm_runner.rs

```diff
@@ -14,31 +18,37 @@ pub struct Counters {
     cnt_steps: u64,
 }
 
+/// Status of the execution of zkAsm program.
+/// Matches json structure returned by zkAsm interpreter.
 #[derive(Deserialize, Debug)]
 pub enum ExecutionStatus {
+    /// Program ran till completion.
     #[serde(rename = "pass")]
     Success,
+    /// Program failed during compilation or execution.
```

Comment:
nit: `RuntimeError` might not be an ideal name if it can include compilation errors as well.

Reply:
True, we would need to rename it in `run-tests-zkasm`, I'll do that in a separate PR.